### DISCUSSION TITLE
ignore files

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#787] [Q&A]
I'm running an index on version 3.18.9 and apparently everything has been processed but still has a single task in "ParsingTask" mode of the Windows.edb file for more than 2 hours.
Is there any way to skip this file?

tks
Paulo

**Comment by lfcnassif** [comment]
Large EDB files can take some time to be processed. Check if there is an esedbexport process running. If yes and if you don't want to wait the decoding of that database, you can kill the esedbexport process.

**Reply by paulobreim** [reply]
thank you
